BigBaseV2/src/core/globals.hpp

```diff
@@ -203,6 +205,8 @@ struct globals {
 		this->vehicle.speedo_meter.y = j["vehicle"]["speedo_meter"]["position_y"];
 
 		this->weapons.custom_weapon = (CustomWeapon)j["weapons"]["custom_weapon"];
+		this->weapons.custom_weapon = j["weapons"]["infinite_ammo"];
+		this->weapons.custom_weapon = j["weapons"]["infinite_mag"];
```

Comment:
You're assigning infinite_ammo and infinite_mag settings to the custom_weapon setting, this will not work.